org.gnome.Rhythmbox3.json

```diff
@@ -1,8 +1,7 @@
 {
     "app-id": "org.gnome.Rhythmbox3",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.34",
-    "default-branch": "stable",
```

Comment:
The default branch is used so that the development branch, and local tests, can use another branch without affecting this one.